pkg/distributor/ha_tracker.go

```diff
@@ -83,6 +59,11 @@ type haTracker struct {
 	electedLock sync.RWMutex
 	elected     map[string]ReplicaDesc // Replicas we are accepting samples from. Key = "user/cluster".
 	clusters    map[string]int         // Number of clusters with elected replicas that a single user has. Key = user.
+
+	electedReplicaChanges         *prometheus.CounterVec
+	electedReplicaTimestamp       *prometheus.GaugeVec
+	electedReplicaPropagationTime prometheus.Histogram
+	kvCASCalls                    *prometheus.CounterVec
```

Comment:
**[nit,non-blocking]:** Would it be possible to move the `haTracker` definition to just above the `newClusterTracker` function? It makes it a bit easier to read when you see the metric definitions and fields near one another.

Reply:
👍 I did that, and also renamed `newClusterTracker` to `newHATracker`.

Reply:
👍